build.assets/charts/Dockerfile-distroless

```diff
@@ -0,0 +1,25 @@
+ARG BASE_IMAGE=gcr.io/distroless/cc-debian11
+
+FROM debian:11 AS staging
+RUN apt-get update 
+COPY fetch-debs ./
+RUN ./fetch-debs dumb-init libpam0g libaudit1 libcap-ng0
```

Comment:
Has the APT repo (and GPG key) been added and updated by this point?

Reply:
I assume so - everything seems to install cleanly.